charts/rancher-vsphere-csi/100.0.1+up2.1.1/Chart.yaml

```diff
@@ -3,15 +3,16 @@ annotations:
   catalog.cattle.io/display-name: vSphere CSI
   catalog.cattle.io/namespace: kube-system
   catalog.cattle.io/os: linux
-  catalog.cattle.io/rancher-version: '>= 2.6.0-0 <= 2.6.99-0'
+  catalog.cattle.io/rancher-version: '>= 2.6.0-0 <= 2.6.100-0'
+  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.20.0-0'
   catalog.cattle.io/release-name: vsphere-csi
 apiVersion: v1
 appVersion: 2.1.1
 description: vSphere Cloud Storage Interface (CSI)
 icon: https://charts.rancher.io/assets/logos/vsphere-csi.svg
 keywords:
 - infrastructure
-kubeVersion: <= 1.19
+kubeVersion: '>= 1.16.0-0 < 1.20.0-0'
```

Comment:
should not be narrowing kubeVersion after-the-fact, you can leave the change to `'< 1.20.0-0'` since that broadens the scope, but remove the check on 1.16 since that's a hard break in functionality.

The annotation having the suggested versions is sufficient enough.